src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/CodeStyle/VisualBasicCodeStyleOptions.vb

```diff
@@ -53,10 +53,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeStyle
                 editorConfigName:="visual_basic_style_unused_value_assignment_preference",
                 defaultValue:=New CodeStyleOption2(Of UnusedValuePreference)(UnusedValuePreference.UnusedLocalVariable, NotificationOption2.Suggestion),
                 optionsBuilder:=s_allOptionsBuilder)
+
+        Public Shared ReadOnly ParenthesesForInvocations As [Option2](Of CodeStyleOption2(Of Boolean)) = CreateOption(
+            VisualBasicCodeStyleOptionGroups.Parentheses, NameOf(ParenthesesForInvocations),
+            defaultValue:=CodeStyleOptions2.FalseWithSuggestionEnforcement, 'Suggestion or silent??
+            EditorConfigStorageLocation.ForBoolCodeStyleOption("visual_basic_style_add_parentheses_for_invocations")) ' RoamingProfileStorageLocation ??
+
+        Public Shared ReadOnly ParenthesesForProperties As [Option2](Of CodeStyleOption2(Of Boolean)) = CreateOption(
+            VisualBasicCodeStyleOptionGroups.Parentheses, NameOf(ParenthesesForProperties),
+            defaultValue:=CodeStyleOptions2.FalseWithSuggestionEnforcement, 'Suggestion or silent??
+            EditorConfigStorageLocation.ForBoolCodeStyleOption("visual_basic_style_remove_parentheses_for_properties")) ' RoamingProfileStorageLocation ??
     End Class
 
     Friend NotInheritable Class VisualBasicCodeStyleOptionGroups
         Public Shared ReadOnly Modifier As New OptionGroup(CompilerExtensionsResources.Modifier_preferences, priority:=1)
         Public Shared ReadOnly ExpressionLevelPreferences As New OptionGroup(CompilerExtensionsResources.Expression_level_preferences, priority:=2)
+        ' I didn't create Parentheses_preferences in resources. Not sure if I shouldn't use it, not also sure where does this value get used? Not sure about priority also :/
+        Public Shared ReadOnly Parentheses As New OptionGroup(CompilerExtensionsResources.Parentheses_preferences, priority:=1)
```

Comment:
i believe this is for determining where to put thse things when we emit .editorconfig.

Reply:
> 
> 
> i believe this is for determining where to put thse things when we emit .editorconfig.

So, it's just about the order of options in the auto-generated .editorconfig?

If so, I'm not sure if you have some rules regarding the orders. I don't know what priority to use.